addons/rook/1.4.3/operator/config-override.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-config-override
+  namespace: rook-ceph
+data:
+  config: |
+    [global]
+    osd pool default size = 1
```

Comment:
default starts at 3, which causes a warning on singe node clusters. Ekco manages all pool sizes so the default is not important.